pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -51,6 +51,18 @@
 DocumentFormat: TypeAlias = Literal['csv', 'doc', 'docx', 'html', 'md', 'pdf', 'txt', 'xls', 'xlsx']
 VideoFormat: TypeAlias = Literal['mkv', 'mov', 'mp4', 'webm', 'flv', 'mpeg', 'mpg', 'wmv', 'three_gp']
 
+# OpenTelemetry GenAI finish reasons used for gen_ai.response.finish_reasons
+# See mappings in provider implementations (e.g., OpenAI/Google) for how vendor reasons map here.
+OtelFinishReason: TypeAlias = Literal[
```

Comment:
Please rename this `FinishReason`

Reply:
Hmm, https://opentelemetry.io/docs/specs/semconv/registry/attributes/gen-ai/ doesn't list any specific finish reasons anymore :/

It used to when I wrote this comment back in June: https://github.com/pydantic/pydantic-ai/pull/1882#discussion_r2122293633

@alexmojaki Does that mean we can just use any string value (i.e. the raw value we get from the API)?

Reply:
They were in https://github.com/open-telemetry/semantic-conventions/blob/v1.36.0/docs/gen-ai/gen-ai-events.md, those events have been removed. Now they're in https://opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-output-messages.json, same values:

```json
{
    "$defs": {
        "FinishReason": {
            "description": "Represents the reason for finishing the generation.",
            "enum": [
                "stop",
                "length",
                "content_filter",
                "tool_call",
                "error"
            ],
            "title": "FinishReason",
            "type": "string"
        },
```